### PR TITLE
73 install husky for lint before commit

### LINKED ISSUE #73: install husky for lint before commit

### PR DESCRIPTION
**Pull request by sergiobtos**
-Installed "husky": "^8.0.0" library
-pre-commit file change to run "npm run lint" before every commit
-Test 3 times, it looks like it is working

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```
Comment (rmc3408) [inline .husky/pre-commit]: add testing too!!

Comment (rmc3408) [reply .husky/pre-commit]: sorry, i did testing husky!!

**Review by rmc3408 — APPROVED** [review]
I did test too! works fine

File: .husky/pre-commit (hunk 1 above)
Comment (sergiobtos) [reply .husky/pre-commit:4]: do you mean this : npm run lint && npm run test  ?

**Pull request merged**